src/components/toast/toast.scss

```diff
@@ -49,6 +50,13 @@ md-toast {
     transition: $swift-ease-out;
 
     @include rtl(justify-content, flex-start, flex-end);
+
+    span {
+      // Layout  [flex]
```

Comment:
may I should use a new mixin   `@include layout_flex()` to centralize this logic?